src/devices/SensorHub/SensorHub.cs

```diff
@@ -0,0 +1,183 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using UnitsNet;
+
+namespace Iot.Device.SensorHub
+{
+    /// <summary>
+    /// SensorHub
+    /// </summary>
+    public class SensorHub
+    {
+        /// <summary>
+        /// Default I2C bus id
+        /// </summary>
+        public const int DefaultI2cBusId = 1;
+
+        /// <summary>
+        /// Default I2C address
+        /// </summary>
+        public const int DefaultI2cAddress = 0x17;
+
+        private readonly I2cDevice _i2cDevice;
+
+        /// <summary>
+        /// Creates a SensorHub with the default bus and connection values
+        /// </summary>
+        /// <returns>SensorHub</returns>
+        public static SensorHub Create()
+        {
+            return new(I2cDevice.Create(new I2cConnectionSettings(DefaultI2cBusId, DefaultI2cAddress)));
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SensorHub"/> class.
+        /// </summary>
+        public SensorHub(I2cDevice i2cDevice)
+        {
+            _i2cDevice = i2cDevice;
+        }
```

Comment:
Is there a way to check that the device is existing?

Reply:
checked for null device like in the other bindings. I don't think there's another way to check it's working

Reply:
I think what @Ellerbach means is that it would be good if you tried to read any of the sensors to find out whether the board is there. `I2cDevice.Create()` (and equivalent functions) will not test whether the given address is in use. So the `i2cDevice` here will not be null when the device does not exist.

Reply:
Thanks, it's what I meant :-)

Reply:
Think this is the best way to check for existing device.